### PR TITLE
Add new fields, including nav-related fields

### PR DESCRIPTION
**Pull request by vmallet**

### CODE REVIEW

File: protos/vehicle_data.proto [hunk 1]
```diff
@@ -114,6 +114,18 @@ enum Field {
...
+  MilesToArrival = 109;
+  MinutesToArrival = 110;
+  OriginLocation = 111;
+  DestinationLocation = 112;
+  CarType = 113;
+  Trim = 114;
+  ExteriorColor = 115;
+  RoofColor = 116;
+  ChargePort = 117;
+  ChargePortLatch = 118;
```
Comment (agbpatro) [inline protos/vehicle_data.proto:128]: can you run https://github.com/teslamotors/fleet-telemetry/blob/main/Makefile#L65 to generate python files as well?

Comment (vmallet) [reply protos/vehicle_data.proto:128]: Good call, done!

Comment (agbpatro) [reply protos/vehicle_data.proto:128]: can you update readme as well?
https://github.com/teslamotors/fleet-telemetry/blob/main/README.md#protos

**Review by agbpatro — APPROVED** [review]

**Review by aaronpkahn — APPROVED** [review]

File: protos/vehicle_data.proto (hunk 1 above)
Comment (vmallet) [reply protos/vehicle_data.proto:128]: Ah that's where I saw it; done.

**Pull request merged**